test/adapters.js

```diff
@@ -1,32 +1,41 @@
 'use strict';
 
-api.metatests.test('callbackify: Promise to callback-last', (test) => {
+const metasync = require('..');
+const metatests = require('metatests');
+
+metatests.test('callbackify: Promise to callback-last', (test) => {
 
   const promise = Promise.resolve('result');
-  const callback = api.metasync.callbackify(promise);
+  const callback = metasync.callbackify(promise);
 
   callback((err, value) => {
-    if (err) throw err;
+    if (err) {
+      test.error(err);
```

Comment:
I think this line and the one below can be merged by adding the message to the `test.error()` call. Also, I think the message shouldn't be capitalized. This comment applies to all of the similar cases below.